app/components/primer/beta/auto_complete.rb

```diff
@@ -32,14 +32,41 @@ class AutoComplete < Primer::Component
         Primer::BaseComponent.new(**system_arguments)
       }
 
+      # Customizable input used to search for results.
+      # It is preferred to use this slot sparingly - it will be created by default if not explicity added.
+      #
+      # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
+      renders_one :input, lambda { |**system_arguments|
+        deny_tag_argument(**system_arguments)
+        deny_single_argument(system_arguments[:autofocus], "autofocus is not allowed for accessibility reasons. See https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus#accessibility_considerations for more information.", **system_arguments)
+        deny_single_argument("aria-label", "instead of `aria-label`, include `label_text` and set `is_label_visible` to `false`.", **system_arguments)
+        system_arguments[:id] = @input_id
+        system_arguments[:name] = @input_id
+        system_arguments[:tag] = :input
+        system_arguments[:autocomplete] = "off"
+
+        system_arguments[:type] = :text
+        system_arguments[:classes] = class_names(
+          "form-control",
+          system_arguments[:classes]
+        )
+
+        Primer::BaseComponent.new(**system_arguments)
+      }
+
       # @example Default
       #   <%= render(Primer::Beta::AutoComplete.new(label_text: "Fruits", src: "/auto_complete", input_id: "fruits-input-1", list_id: "fruits-popup-1", position: :relative)) %>
       #
       # @example With Non-Visible Label
       #   <%= render(Primer::Beta::AutoComplete.new(label_text: "Fruits", src: "/auto_complete", input_id: "fruits-input-2", list_id: "fruits-popup-2", is_label_visible: false, position: :relative)) %>
       #
-      # @example With Custom Classes for the Results
+      # @example With Custom Classes for the Input
       #   <%= render(Primer::Beta::AutoComplete.new(label_text: "Fruits", src: "/auto_complete", input_id: "fruits-input-3", list_id: "fruits-popup-3", position: :relative)) do |c| %>
+      #     <% c.input(classes: "special-form", autofocus: true) %>
```

Comment:
should we remove `autofocus: true` in this example given the guidance we provide?

> "autofocus is not allowed for accessibility reasons"

Reply:
Yes! Sorry I added this to test if the `deny_single_argument` tags were working (looks like this is outdated now anyway!)